proto/cometbft/rpc/grpc/v2/types.proto

```diff
@@ -16,6 +16,7 @@ message ResponseBroadcastTx {
 //----------------------------------------
 // Service Definition
 
+// This service has been removed after CometBFT 0.37.
```

Comment:
It's actually after v0.38. It was just deprecated in v0.38: https://github.com/cometbft/cometbft/blob/3b7c719fc892f011e602e906b36b9a56edc60bee/proto/tendermint/rpc/grpc/types.proto#L32

Reply:
OK, I have restored the APIs complete with the generated `*.pb.go` files. They are not used anywhere else in the code.